platforms/custom-s3.md

```diff
@@ -2,4 +2,37 @@
 
 This defines the S3 interface for providers other than AWS (e.g. minio-based).
 
-- `platform`: The API URL, must be the endpoint URL that can be used for the AWS CLI for example.
+- `platform`: The API URL (template), must be the endpoint URL that can be used for the AWS CLI for example, e.g. `https://{bucket}.example.com` or `http://example.com:9000`.
+- `bucket`: The bucket name, if applicable.
+- `region`: The region, if applicable.
+
+## Mapping to S3 tooling
+
+### GDAL (`/vsis3/`)
+
+GDAL documentation: <https://gdal.org/en/latest/user/virtual_file_systems.html#vsis3-aws-s3-files>
+
+- `platform`: Some options for S3 can be inferred from the given URL (template):
+  - `AWS_HTTPS` can be retrieved by parsing the scheme part of the URL. `https` = `ON`, `http` = `OFF`.
+  - `AWS_S3_ENDPOINT` is the authority part of the URL after replacing all variables in the URL.
+  - `AWS_VIRTUAL_HOSTING` must be set to `FALSE` if there's no `{bucket}` placeholder in the URL template, otherwise `TRUE` (default value).
+- The `region` property corresponds to the `AWS_REGION` option.
+- The `requester_pays` property corresponds to the `AWS_REQUEST_PAYER` option. If `requester_pays` is `true`, set `AWS_REQUEST_PAYER` to `requester`.
+- If `s3` exists in `auth:refs`, you should set `AWS_NO_SIGN_REQUEST` to `YES`. Otherwise it should be `NO`.
```

Comment:
If `s3` exists in `auth:refs`, the `AWS_NO_SIGN_REQUEST` should be set to `NO` not to `YES`.

Reply:
```suggestion
- If the `s3` authentication method is referred to through `auth:refs`, you should set `AWS_NO_SIGN_REQUEST` to `NO`. Otherwise it should be `YES`.
```